docs/THIRDPARTY

```diff
@@ -253,6 +253,12 @@ Copyright: Google, Inc.
 License: Apache-2.0
 Comment: Material volume up icon (rounded)
 
+Files: web/shared/icons/recent.svg
+Source: https://lucide.dev/icons/history
+Copyright: 2013-2022 Cole Bemis
+License: ISC License
+Comment: history icon
```

Comment:
Probably no need for a Comment here; see other entries in this fails for how we typically use that field.